### PR TITLE
Makes space dust event less invisible.

### LINKED ISSUE #17704: Randomly fucking crit while in space
I've seen this happen 3 times to myself and its been reported by 2 others.

You're in space around the station, then out of nowhere you're fucking crit.

There's no projectile, no indication, no message about what's struck you. Every time its 100% out of the blue with absolutely no hint or suggestion available to players OR ADMINS as to what has happened.

Some people suggest its space dust, I haven't looked at our space dust code but if space dust is literally just a "crits random people in space" mechanic, it needs to go.

### PR DESCRIPTION
**Pull request by Supermichael777**
It was very common, quite lethal, almost completely silent and never gave any indication you were being hit other then a tiny click and 30 damage. Its now a lot less common, Louder, and tells admins that the station is taking random potshots from a machine-gun

Fixes #17704 by making it more obvious that your being shot by RNJesus

### CODE REVIEW

**Comment by Shadowlight213** [comment]
Space dust actually happens? I've like rarely ever seen it.
If you are nerfing its frequency, could you please buff its ability to damage the hull?

**Comment by Armhulen** [comment]
wait do you feel the freeze????

**Comment by coiax** [comment]
While you're there, make being hit by an asteroid directly give the person a message.

**Comment by RandomMarine** [comment]
Space dust happens. Often punches holes in the station, too. Most times I see random holes in maint, or the HoS office or permabrig windows broken, it's fukken dust.

File: code/game/gamemodes/meteor/meteors.dm [hunk 1]
```diff
@@ -137,6 +137,7 @@
 	//first bust whatever is in the turf
 	for(var/atom/A in T)
 		if(A != src)
+			A.visible_message("<span class='warning'>[src] slams into [A].</span>", "<span class='userdanger'>[src] slams into you!.</span>")
```
Comment (phil235) [inline code/game/gamemodes/meteor/meteors.dm]: mob/visible_message() and atom/proc/visible_message() do not take the same arguments and in the same order.

File: code/modules/events/dust.dm [hunk 2]
```diff
@@ -1,10 +1,10 @@
 /datum/round_event_control/meteor_wave/dust
 	name = "Minor Space Dust"
 	typepath = /datum/round_event/meteor_wave/dust
-	weight = 200
+	weight = 50		// A lot, but not so much that it’s a constant hazard.
```
Comment (Dorsisdwarf) [inline code/modules/events/dust.dm]: No No No No No No No

Space dust weights out all the other events you madman

Comment (Supermichael777) [inline code/modules/events/dust.dm]: but it doesnt make the game FUN<sup>TM</sup>

Comment (coiax) [inline code/modules/events/dust.dm]: I have a feeling that if you lower the weight of space dust, that will, by proxy, increase the chances of everything else.

Comment (PKPenguin321) [inline code/modules/events/dust.dm]: :-1: to this part specifically

Comment (Dorsisdwarf) [inline code/modules/events/dust.dm]: That is, in fact, how it works (or at least how I was informed it works)

File: code/modules/events/dust.dm [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /datum/round_event_control/meteor_wave/dust
 	name = "Minor Space Dust"
 	typepath = /datum/round_event/meteor_wave/dust
-	weight = 200
+	weight = 50		// A lot, but not so much that it’s a constant hazard.
 	max_occurrences = 1000
-	earliest_start = 0
-	alertadmins = 0
+	earliest_start = 3000	// Five minuets 
+	alertadmins = 1 	// This is surprisingly lethal. Plus no announcement.
```
Comment (KorPhaeron) [inline code/modules/events/dust.dm]: Please no, we don't care about this

Comment (coiax) [inline code/modules/events/dust.dm]: So the PR will literally become "space dust has a louder noise on impact".

**Comment by coiax** [comment]
Also, if it doesn't start until 5 minutes, that just makes all the other events fire more in the first five minutes.

**Comment by LatD** [comment]
Space dust is one of those things you never see when you play but see every round when you observe

File: code/modules/events/dust.dm (hunk 3 above)
Comment (KorPhaeron) [inline code/modules/events/dust.dm]: That'd be ideal yes

**Comment by Bawhoppen** [comment]
this does NOT need a changelog
god why is everyone so changelog-happy nowadays

**Comment by PKPenguin321** [comment]
it's mainly just coiax, he treats the changelog like a pageant child

**Comment by Jordie0608** [comment]
Go away Bawhoppen stop undoing all my efforts

**Comment by xxalpha** [comment]
Anything relevant to the players needs a changelog.

File: code/game/gamemodes/meteor/meteors.dm (hunk 1 above)
Comment (Supermichael777) [inline code/game/gamemodes/meteor/meteors.dm]: Istype incoming

Comment (KorPhaeron) [inline code/game/gamemodes/meteor/meteors.dm]: no stop

File: code/game/gamemodes/meteor/meteors.dm [hunk 4]
```diff
@@ -190,7 +192,7 @@
 	pass_flags = PASSTABLE | PASSGRILLE
 	hits = 1
 	hitpwr = 3
-	meteorsound = 'sound/weapons/throwtap.ogg'
+	meteorsound = 'sound/weapons/Gunshot_smg.ogg'
```
Comment (KorPhaeron) [inline code/game/gamemodes/meteor/meteors.dm]: odd choice

Comment (Supermichael777) [inline code/game/gamemodes/meteor/meteors.dm]: Kind of a placeholder to replace throw-tap without reverting to the station rocking boom. hard to find a noise that fist everything space dust hits from metal to glass to Robusten.

File: code/game/gamemodes/meteor/meteors.dm [hunk 5]
```diff
@@ -137,6 +137,8 @@
 	//first bust whatever is in the turf
 	for(var/atom/A in T)
 		if(A != src)
+			if(istype(A, /mob))
```
Comment (phil235) [inline code/game/gamemodes/meteor/meteors.dm]: ghosts and ai eye camera?

**Comment by Supermichael777** [comment]
After the freeze i'm adding bluespace meteors.

**Pull request merged**